src/astro/utils/load.py

```diff
@@ -98,38 +99,43 @@ def flatten_ndjson(
 
 
 def populate_normalize_config(
```

Comment:
Should we move this to `sql/operators/load_file` ? And what about other functions and methods in this file, are they needed? Or can they be deleted

Reply:
We can certainly move this to load_file. I'll check for other functions as I'm not sure about their usage.

Reply:
I looked up, other methods of load files, they are mainly used in sql_dataframe.py and sql_decorator.py, which I think Daniel PR should cover.